packages/node/src/edge-wasm-plugin.ts

```diff
@@ -0,0 +1,89 @@
+import type { Plugin } from 'esbuild';
```

Comment:
Is this file copy-pasted from the other place where we support this? Should we share this logic directly somewhere?

Reply:
Nope but it is very similar to what we do in Next.js and when deploying edge functions

Reply:
Can you add a link to the source code in a comment?

Reply:
I tried looking for the original, but couldn't find it. I asked around for someone to point it out.

Reply:
I added a comment at the top of this file linking to the original.